contracts/dev-utils/contracts/src/LibDydxBalance.sol

```diff
@@ -0,0 +1,418 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.16;
+pragma experimental ABIEncoderV2;
+
+import "@0x/contracts-asset-proxy/contracts/src/interfaces/IAssetData.sol";
+import "@0x/contracts-asset-proxy/contracts/src/interfaces/IDydxBridge.sol";
+import "@0x/contracts-asset-proxy/contracts/src/interfaces/IDydx.sol";
+import "@0x/contracts-erc20/contracts/src/LibERC20Token.sol";
+import "@0x/contracts-exchange-libs/contracts/src/LibOrder.sol";
+import "@0x/contracts-utils/contracts/src/LibBytes.sol";
+import "@0x/contracts-utils/contracts/src/LibSafeMath.sol";
+import "./LibAssetData.sol";
+import "./D18.sol";
+
+
+library LibDydxBalance {
+
+    using LibBytes for bytes;
+    using LibSafeMath for uint256;
```

Comment:
Nit - `using D18 for uint256` and `using D18 for int256` could improve readability and consistency w other code. But it's not too bad so up to you 🤷‍♂.

Reply:
Actually, I kind of like the long-hand since it makes it clearer that these `uint256`s are expressed in a different unit.